tests/visualization/positioning_test.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import torch
+
+from adam.visualization.positioning import (
+    AxisAlignedBoundingBox,
+    main,
+    CollisionPenalty,
+    DistanceFromOriginPenalty,
+)
+
+
+def test_positioning() -> None:
+    np.random.seed(127)
+    main()
+
+    # positioned to be colliding with one another:
+    aabb0 = AxisAlignedBoundingBox.create_at_center_point(
+        center=np.ndarray(shape=(3,), buffer=np.array([0, 0, 0]))
+    )
+    aabb1 = AxisAlignedBoundingBox.create_at_center_point(
+        center=np.ndarray(shape=(3,), buffer=np.array([0.5, 0.5, 0.5]))
+    )
+
+    # set to definitely not be near the origin:
+    aabb_rand = AxisAlignedBoundingBox.create_at_random_position(
```

Comment:
For a test, rather than creating at random positions which are known not to collide, it would be better to create the boxes explicitly at colliding and non-colliding positions.